.husky/scripts/dont-trust.sh

```
@@ -0,0 +1,19 @@
+# Verify if the commit is signed, why not?
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)